### PR TITLE
Add design doc for a manual annotation tool

### PR DESCRIPTION
**Pull request by kabilar**
- [x] After discussion with Anastasia and Hong-Hsi, we have defined 2 use cases.
- [x] I have collated the requirements from the milestones final document and the above use cases.  My local copy of this design doc matches each requirement to a milestone number.  I have not pushed these milestone numbers to GitHub since we probably don't want specific information on the milestones to be public.
- [ ] Evaluate possible solutions and add a description of the evaluations to this document

cc @aaronkanzer @ayendiki @satra

### CODE REVIEW

File: doc/design/linc-annotation.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+7. Team-based augmentation of labeled datasets
+8. Add landmarks
+9. Pixel-wise classifier (i.e. trace individual axons)
+10. Create other labels used to evaluate automated methods - Need to specify further.
+11. Make annotations available for dissemination
+
+## Open questions
+
+1. What storage format should be used for the two different annotation types?  Zarr?
+
```
Comment (kabilar) [inline doc/design/linc-annotation.md:40]:
```suggestion
1. What storage format should be used for the two different annotation types?  Zarr?
2. How are the annotations stored?  In a bucket or database within the container?
```

**Comment by aaronkanzer** [comment]
@kabilar do we want to close this now that we've gone with WebKNOSSOS for the short-to-medium term?

**Comment by kabilar** [comment]
> @kabilar do we want to close this now that we've gone with WebKNOSSOS for the short-to-medium term?

Thanks @aaronkanzer.  I will work on updating this design doc and merging it in so that we have it for future reference.

**Comment by kabilar** [comment]
Closing this pull request as I am re-forking `dandi/dandi-archive` directly to `kabilar`.  Will reopen a new pull request.

**Pull request closed without merging**